src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -329,8 +328,8 @@ protected override void ApplyResponseChallenge()
                 return;
             }
 
```

Comment:
Filed https://github.com/aspnet/Security/issues/166 to deal with this in a different PR